polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaSource.scala

```diff
@@ -130,9 +130,13 @@ class ScalaSource[Interpreter <: ScalaInterpreter](val interpreter: Interpreter)
       import global.Quasiquote
       trees.foldLeft((Vector.empty[Tree], Vector.empty[Tree])) {
         case ((moduleTrees, classTrees), tree) => tree match {
-          case tree: global.ValDef if !tree.mods.hasFlag(global.Flag.PRIVATE) && !tree.mods.hasFlag(global.Flag.PROTECTED) =>
-            // make a lazy val accessor in the companion
-            (moduleTrees :+ q"lazy val ${tree.name} = INSTANCE.${tree.name}", classTrees :+ tree)
+          case tree: global.ValDef =>
```

Comment:
Oops... I think I figured private and protected ValDefs would get handled by the catch-all case, but they were handled instead by the `MemberDef` case, which would be bad.